### ISSUE TITLE
Val file is not included in the Dataset provided

### ISSUE THREAD

**Issue by victorabilash** [shervinmin/DeepCovid#5]
**_FileNotFoundError: [WinError 3] The system cannot find the path specified: './data/val'_**

Since Val folder is not included in the provided dataset, the program practically fails to go ahead from the building stage.
Do update the dataset with a valid Val folder.

**Comment by lizhenstat** [comment]
@victorabilash 
Hi, same question here and I send email  to the author and the author replies:
      "If I remember correctly, I used 10% of the training data as the validation set, so you many want to do the same thing."
